### ISSUE TITLE
addDatabaseQuery does not increase queries count

### ISSUE THREAD

**Issue by MarkusJLechner** [itsgoingd/clockwork#703]
We have converted a legacy project to Laravel and still have many vanilla DB fetches.

We created a wrapper above PDOStatement to log all queries, excluding those made by Laravel's query builder. This works well, but I noticed that the total query count is incorrect. It only counts when the Laravel query event is triggered, but not when calling `clock()->addDatabaseQuery` as described in the docs.

Is there a way to increase the count? Instead of using `addDatabaseQuery`, I tried manually triggering the Laravel query event, but then I cannot pass options like trace skipping.

Any advice on how to properly increase the count?

**Comment by itsgoingd** [comment]
Hey, it would totally make sense if the `addDatabaseQuery` helper incremented query counts, I'll look into adding it for the next release.

As a workaround for now, you can increment the database queries yourself in your own wrapper like this:

```php
$request = clock()->request();

// Increment the total query count
$request->databaseQueriesCount++;

// Increment query count of a particular type
$request->databaseSelects++;
$request->databaseInserts++;
$request->databaseUpdates++;
$request->databaseDeletes++;
$request->databaseOthers++;
```

**Comment by itsgoingd** [comment]
This is now implemented in master (https://github.com/itsgoingd/clockwork/commit/bbcefdf1e32540e9a2de5fbd1eacbb77d3f50bda) and will be released in Clockwork 5.3.
